### PR TITLE
[FLINK-19478][docs-zh][python]  Translate page "intro_to_table_api" into Chinese

### PR DESCRIPTION
**Pull request by wangxlong**
## What is the purpose of the change

 Translate page "intro_to_table_api" into Chinese

## Brief change log

 Translate page "intro_to_table_api" into Chinese

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (no)
  - The serializers: (no)
  - The runtime per-record code paths (performance sensitive): (no)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn/Mesos, ZooKeeper: (no)
  - The S3 file system connector: (no)

## Documentation

  - Does this pull request introduce a new feature? (no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

**Comment by wangxlong** [comment]
Hi @dianfu  Could you help to have a review in your free time, Thank you~

File: docs/dev/python/table-api-users-guide/intro_to_table_api.zh.md [hunk 1]
```diff
@@ -384,32 +384,32 @@ The result is:
...
-The `Table` objects used in Table API and the tables used in SQL can be freely converted to each other.
+Table API 中的 `Table` 对象和 SQL 中的 tables 可以自由地相互转换。
 
-The following example shows how to use a `Table` object in SQL:
+下面例子展示了如何在 SQL 中使用 `Table` 对象：
 
 {% highlight python %}
 
-# create a sink table to emit results
+# 创建一张 sink 表来发送数据
```
Comment (dianfu) [inline docs/dev/python/table-api-users-guide/intro_to_table_api.zh.md:412]:
```suggestion
# 创建一张 sink 表来接收结果数据
```

File: docs/dev/python/table-api-users-guide/intro_to_table_api.zh.md [hunk 2]
```diff
@@ -419,27 +419,27 @@ table_env.execute_sql("""
     )
 """)
 
-# convert the Table API table to a SQL view
+# 将 Table API 表转换成 SQL 的视图
 table = table_env.from_elements([(1, 'Hi'), (2, 'Hello')], ['id', 'data'])
 table_env.create_temporary_view('table_api_table', table)
 
-# emit the Table API table
+# 将数据写入 Table API 表
```
Comment (dianfu) [inline docs/dev/python/table-api-users-guide/intro_to_table_api.zh.md:426]:
```suggestion
# 将 Table API 表的数据写入结果表
```

File: docs/dev/python/table-api-users-guide/intro_to_table_api.zh.md [hunk 3]
```diff
@@ -455,18 +455,18 @@ table_env.execute_sql("""
     )
 """)
 
-# convert the sql table to Table API table
+# 将 sql 表转化成 Table API 表
```
Comment (dianfu) [inline docs/dev/python/table-api-users-guide/intro_to_table_api.zh.md:458]:
```suggestion
# 将 sql 表转换成 Table API 表
```

File: docs/dev/python/table-api-users-guide/intro_to_table_api.zh.md [hunk 4]
```diff
@@ -455,18 +455,18 @@ table_env.execute_sql("""
...
 table = table_env.from_path("sql_source")
 
-# or create the table from a sql query
+# 或者从 sql 查询语句中创建表
 table = table_env.sql_query("SELECT * FROM sql_source")
 
-# emit the table
+# 将数据发送到该表中
```
Comment (dianfu) [inline docs/dev/python/table-api-users-guide/intro_to_table_api.zh.md:464]:
```suggestion
# 将表中的数据写出
```

File: docs/dev/python/table-api-users-guide/intro_to_table_api.zh.md [hunk 5]
```diff
@@ -492,20 +492,20 @@ table.to_pandas()
...
 
 {% highlight text %}
    id   data
 0   1     Hi
 1   2  Hello
 {% endhighlight %}
 
-<span class="label label-info">Note</span> "to_pandas" will trigger the materialization of the table and collect table content to the memory of the client, it's good practice to limit the number of rows collected via <a href="{{ site.pythondocs_baseurl }}/api/python/pyflink.table.html#pyflink.table.Table.limit">Table.limit</a>.
-<span class="label label-info">Note</span> "to_pandas" is not supported by the flink planner, and not all data types can be emitted to pandas DataFrames.
+<span class="label label-info">Note</span> "to_pandas" 将会触发表的物化同时将表的内容收集到客户端内存中，所以通过 <a href="{{ site.pythondocs_baseurl }}/api/python/pyflink.table.html#pyflink.table.Table.limit">Table.limit</a> 来限制收集数据的条数是一种很好的做法。
```
Comment (dianfu) [inline docs/dev/python/table-api-users-guide/intro_to_table_api.zh.md:503]:
```suggestion
<span class="label label-info">Note</span> "to_pandas" 将会触发表的物化，同时将表的内容收集到客户端内存中，所以通过 <a href="{{ site.pythondocs_baseurl }}/api/python/pyflink.table.html#pyflink.table.Table.limit">Table.limit</a> 来限制收集数据的条数是一种很好的做法。
```

File: docs/dev/python/table-api-users-guide/intro_to_table_api.zh.md [hunk 6]
```diff
@@ -539,13 +539,13 @@ table_env.execute_sql("INSERT INTO sink_table SELECT * FROM table_source").wait(
 
 {% endhighlight %}
 
-### Emit Results to Multiple Sink Tables
+### 将数据发送到多张的 Sink 表中
```
Comment (dianfu) [inline docs/dev/python/table-api-users-guide/intro_to_table_api.zh.md:542]:
```suggestion
### 将数据发送到多张 Sink 表中
```

File: docs/dev/python/table-api-users-guide/intro_to_table_api.zh.md [hunk 7]
```diff
@@ -565,21 +565,21 @@ table_env.execute_sql("""
     )
 """)
 
-# create a statement set
+# 创建 statement set
 statement_set = table_env.create_statement_set()
 
-# emit the "table" object to the "first_sink_table"
+# 发送 "table" 对象到 "first_sink_table"
```
Comment (dianfu) [inline docs/dev/python/table-api-users-guide/intro_to_table_api.zh.md:571]:
```suggestion
# 将 "table" 的数据写入 "first_sink_table"
```

File: docs/dev/python/table-api-users-guide/intro_to_table_api.zh.md [hunk 8]
```diff
@@ -565,21 +565,21 @@ table_env.execute_sql("""
...
 statement_set.add_insert("first_sink_table", table)
 
-# emit the "simple_source" to the "second_sink_table" via a insert sql query
+# 通过一条 sql 插入查询语句将数据从 "simple_source" 发送到 "second_sink_table"
```
Comment (dianfu) [inline docs/dev/python/table-api-users-guide/intro_to_table_api.zh.md:574]:
```suggestion
# 通过一条 sql 插入语句将数据从 "simple_source" 发送到 "second_sink_table"
```

File: docs/dev/python/table-api-users-guide/intro_to_table_api.zh.md [hunk 9]
```diff
@@ -588,23 +588,23 @@ The result is:
 7> +I(2,Hello)
 {% endhighlight %}
 
-Explain Tables
+Explain 表
 -----------------
 
-The Table API provides a mechanism to explain the logical and optimized query plans used to compute a `Table`. 
-This is done through the `Table.explain()` or `StatementSet.explain()` methods. `Table.explain()`returns the plan of a `Table`. `StatementSet.explain()` returns the plan for multiple sinks. These methods return a string describing three things:
+Table API 提供了一种机制来翻译 `Table` 的逻辑和优化查询计划。
```
Comment (dianfu) [inline docs/dev/python/table-api-users-guide/intro_to_table_api.zh.md:594]:
```suggestion
Table API 提供了一种机制来解释 `Table` 的逻辑查询计划和优化后的查询计划。 
```

File: docs/dev/python/table-api-users-guide/intro_to_table_api.zh.md [hunk 10]
```diff
@@ -588,23 +588,23 @@ The result is:
...
 {% endhighlight %}
 
-Explain Tables
+Explain 表
 -----------------
 
-The Table API provides a mechanism to explain the logical and optimized query plans used to compute a `Table`. 
-This is done through the `Table.explain()` or `StatementSet.explain()` methods. `Table.explain()`returns the plan of a `Table`. `StatementSet.explain()` returns the plan for multiple sinks. These methods return a string describing three things:
+Table API 提供了一种机制来翻译 `Table` 的逻辑和优化查询计划。 
+这个通过 `Table.explain()` 或者 `StatementSet.explain()` 方法来完成。`Table.explain()` 返回 `Table` 的执行计划。`StatementSet.explain()` 返回多个 sink 的执行计划。这些方法返回一个字符串来描述三个方面：
```
Comment (dianfu) [inline docs/dev/python/table-api-users-guide/intro_to_table_api.zh.md:595]:
```suggestion
这是通过 `Table.explain()` 或者 `StatementSet.explain()` 方法来完成的。`Table.explain()` 可以返回一个 `Table` 的执行计划。`StatementSet.explain()` 则可以返回含有多个 sink 的作业的执行计划。这些方法会返回一个字符串，字符串描述了以下三个方面的信息：
```

File: docs/dev/python/table-api-users-guide/intro_to_table_api.zh.md [hunk 11]
```diff
@@ -64,84 +64,84 @@ table_env.execute_sql("""
     )
 """)
 
-# 4. query from source table and perform caculations
-# create a Table from a Table API query:
+# 4. 从 source 表中查询同时执行计算
```
Comment (dianfu) [inline docs/dev/python/table-api-users-guide/intro_to_table_api.zh.md:67]:
```suggestion
# 4. 查询 source 表，同时执行计算
```

File: docs/dev/python/table-api-users-guide/intro_to_table_api.zh.md [hunk 12]
```diff
@@ -64,84 +64,84 @@ table_env.execute_sql("""
...
+# 使用 Table API 查询语句创建一张表：
```
Comment (dianfu) [inline docs/dev/python/table-api-users-guide/intro_to_table_api.zh.md:68]:
```suggestion
# 通过 Table API 查询语句创建一张表：
```
使用 -> 通过？其他地方也看一下是否有必要改动。

**Review by dianfu — COMMENTED** [review]
@wangxlong Thanks for the PR. Have left a few comments.

File: docs/dev/python/table-api-users-guide/intro_to_table_api.zh.md (hunk 12 above)
Comment (wangxlong) [reply docs/dev/python/table-api-users-guide/intro_to_table_api.zh.md]: `使用`改为`通过`是会通顺些，我看下下其它地方，可以改动的都改动了。

**Comment by wangxlong** [comment]
@dianfu Updated.

**Pull request closed without merging**